python-frontend/src/main/java/org/sonar/python/semantic/v2/TypeInferenceV2.java

```diff
@@ -64,7 +65,7 @@ public TypeInferenceV2(ProjectLevelTypeTable projectLevelTypeTable) {
 
   @Override
   public void visitFileInput(FileInput fileInput) {
-    var type = new ModuleType("somehow get its name", new HashMap<>());
+    var type = new ModuleType("somehow get its name");
```

Comment:
Can you maybe create a small ticket for this?
I think basically we need to get the `PythonFile` here to fix this.

Reply:
Good catch, I've created the [SONARPY-1778](https://sonarsource.atlassian.net/browse/SONARPY-1778)